src/pool/srv_pool.c

```diff
@@ -3722,12 +3798,12 @@ ds_pool_update(uuid_t pool_uuid, crt_opcode_t opc,
 	int				rc;
 	char				*env;
 
-	/* Convert target address list to target id list */
 	rc = pool_find_all_targets_by_addr(pool_uuid, list, &target_list,
-					   out_list, hint);
+			out_list, hint);
```

Comment:
DAOS coding style wants the second line aligned with the opening params. Should revert.